### PR TITLE
[Doc]translate doc for docs/zh_cn/user_guides/5_deployment.md

### PR DESCRIPTION
**Pull request by ooooo-create**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

translate doc for docs/zh_cn/user_guides/5_deployment.md

## Modification

update `docs/en/user_guides/5_deployment.md`
fix `docs/zh_cn/user_guides/5_deployment.md`

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMDet3D.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by ooooo-create** [comment]
@AI-Tianlong 麻烦帮我review一下

**Comment by AI-Tianlong** [comment]
😍

File: docs/en/user_guides/5_deployment.md [hunk 1]
```diff
@@ -1,184 +1,238 @@
...
-    --input-img ${INPUT_IMG} \
-    --shape ${INPUT_SHAPE} \
-    --rescale-shape ${RESCALE_SHAPE} \
-    --show \
-    --verify \
-    --dynamic-export \
-    --cfg-options \
-      model.test_cfg.mode="whole"
-```
+[MMSegmentation](https://github.com/open-mmlab/mmsegmentation/tree/main), also known as `mmseg`, is an open source object segmentation toolbox based on Pytorch. It's a part of the [OpenMMLab](<(https://openmmlab.com/)>) object.
```
Comment (xiexinch) [inline docs/en/user_guides/5_deployment.md:20]:
```suggestion
[MMSegmentation](https://github.com/open-mmlab/mmsegmentation/tree/main), also known as `mmseg`, is an open source semantic segmentation toolbox based on Pytorch. It's part of the [OpenMMLab](<(https://openmmlab.com/)>) object.
```

File: docs/en/user_guides/5_deployment.md [hunk 2]
```diff
@@ -1,184 +1,238 @@
...
+## Installation
+
+### Install mmseg
+
+Please follow the [Installation Guide](https://mmsegmentation.readthedocs.io/en/latest/get_started.html).
+
+### Install mmdeploy
 
-Description of arguments:
+Following installation  for mmdeploy:
```
Comment (xiexinch) [inline docs/en/user_guides/5_deployment.md:30]:
```suggestion
`mmdeploy` can be installed as follows:
```

File: docs/en/user_guides/5_deployment.md [hunk 3]
```diff
@@ -1,184 +1,238 @@
...
-- `--checkpoint` : The path of a model checkpoint file.
-- `--output-file`: The path of output ONNX model. If not specified, it will be set to `tmp.onnx`.
-- `--input-img` : The path of an input image for conversion and visualize.
-- `--shape`: The height and width of input tensor to the model. If not specified, it will be set to img_scale of test_pipeline.
-- `--rescale-shape`: rescale shape of output, set this value to avoid OOM, only work on `slide` mode.
-- `--show`: Determines whether to print the architecture of the exported model. If not specified, it will be set to `False`.
-- `--verify`: Determines whether to verify the correctness of an exported model. If not specified, it will be set to `False`.
-- `--dynamic-export`: Determines whether to export ONNX model with dynamic input and output shapes. If not specified, it will be set to `False`.
-- `--cfg-options`:Update config options.
+**The first way to install:** Install precompiled package
```
Comment (xiexinch) [inline docs/en/user_guides/5_deployment.md:32]:
```suggestion
**Option 1:** Install precompiled package
```

File: docs/en/user_guides/5_deployment.md [hunk 4]
```diff
@@ -1,184 +1,238 @@
...
 
-:::{note}
-This tool is still experimental. Some customized operators are not supported for now.
-:::
+Please follow the [Installation overview](https://mmdeploy.readthedocs.io/zh_CN/latest/get_started.html#mmdeploy)
+
+**The second way to install:**  Automatic Installation script
```
Comment (xiexinch) [inline docs/en/user_guides/5_deployment.md:36]:
```suggestion
**Option 2:**  Automatic Installation script
```

File: docs/en/user_guides/5_deployment.md [hunk 5]
```diff
@@ -1,184 +1,238 @@
...
 
-### Evaluate ONNX model
+**NOTE**:
 
-We provide `tools/deploy_test.py` to evaluate ONNX model with different backend.
+- Add `$(pwd)/build/lib` to `PYTHONPATH`, can loading mmdeploy SDK python package `mmdeploy_runtime`. See [SDK model inference](#SDK-model-inference) for more information.
+- With [ONNX Runtime model inference](#Backend-model-inference), need to load custom operator library and add ONNX Runtime Library's PATH to `LD_LIBRARY_PATH`.
 
-### Prerequisite
+**The third way to install:**  Build MMDeploy from source
```
Comment (xiexinch) [inline docs/en/user_guides/5_deployment.md:54]:
```suggestion
**Option 3:**  Build MMDeploy from source
```

**Review by xiexinch — COMMENTED** [review]
Could you add an instruction to install mmdeploy using mim?

**Comment by ooooo-create** [comment]
> Could you add an instruction to install mmdeploy using mim?

Option 4?

**Comment by ooooo-create** [comment]
> Could you add an instruction to install mmdeploy using mim?

已添加

**Review by xiexinch — APPROVED** [review]

File: docs/en/user_guides/5_deployment.md [hunk 6]
```diff
@@ -1,184 +1,255 @@
...
+```
 
-### Evaluate ONNX model
+2. Install mmdeploy
 
-We provide `tools/deploy_test.py` to evaluate ONNX model with different backend.
+```shell
+git clone https://github.com/open-mmlab/mmdeploy.git
+cd mmdeploy
+mim install -e.
```
Comment (xiexinch) [inline docs/en/user_guides/5_deployment.md:68]:
```suggestion
mim install -e .
```

**Pull request merged**